python/cuml/test/test_random_forest.py

```diff
@@ -171,6 +171,9 @@ def test_rf_classification_default(datatype, column_info, nrows):
     fil_acc = accuracy_score(y_test, fil_preds)
     cu_acc = accuracy_score(y_test, cu_preds)
 
+    score_acc = cuml_model.score(X_test, y_test)
+    assert cu_acc == pytest.approx(score_acc)
```

Comment:
why were this not in the tests before?

Reply:
There was no test for score and unsurprisingly score was broken.
The broken score function prevents hyperopt from working with RF, so I fixed it and expanded the test here.